spec/markdown-preview-view-spec.coffee

```diff
@@ -99,6 +99,17 @@ describe "MarkdownPreviewView", ->
           }
         """
 
+  describe "editor behaves read-only", ->
+    beforeEach ->
+      waitsForPromise ->
+        preview.renderMarkdown()
+
+    describe "when a code block is rendered", ->
+      it "it does not have line-cursor decorations", ->
+        plainEditor = preview.find("atom-text-editor[data-grammar='text plain null-grammar']")
```

Comment:
I'd just call it `editor` since you don't need to contrast it with another, non-plain editor.